### PR TITLE
schema updates for project management

### PR DESCRIPTION
**Pull request by Pomax**
fixes https://github.com/mozilla/publish.webmaker.org/issues/159
fixes https://github.com/mozilla/publish.webmaker.org/issues/163

### CODE REVIEW

**Comment by Pomax** [comment]
I think this might still need a docs edit for the `/projects` route...

**Comment by Pomax** [comment]
I do not understand why this is erroring out... =(

File: seeds/1_users.js [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
 
-    .then(function() { return knex('users').insert({ name: 'ag-dubs' }); })
-    .then(function() { return knex('users').insert({ name: 'k88hudson' }); })
-    .then(function() { return knex('users').insert({ name: 'jbuckca' }); });
+  return Promise.join(
+    knex.insert(users[0]).into('users').catch(function() { return knex.table('users'); }),
+    knex.insert(users[1]).into('users').catch(function() { return knex.table('users'); }),
+    knex.insert(users[2]).into('users').catch(function() { return knex.table('users'); }),
+    knex.insert(users[3]).into('users').catch(function() { return knex.table('users'); })
+  );
```
Comment (Pomax) [inline seeds/1_users.js]: catches are for ignoring duplication errors if `npm run knex` or `npm run test` is run after the database was already built.

Comment (sedge) [inline seeds/1_users.js]: Can you add that in a comment?

Comment (Pomax) [inline seeds/1_users.js]: I can do that.

**Comment by Pomax** [comment]
superceded by https://github.com/mozilla/publish.webmaker.org/pull/171

**Pull request closed without merging**